gpii/node_modules/canopyMatchMaker/src/CanopyMatchMaker.js

```diff
@@ -78,7 +78,8 @@ gpii.canopyMatchMaker.matchPost.handleRequest = function (canopyMatchMaker, requ
  */
 gpii.canopyMatchMaker.match = function (ontologyHandler, payload, disposeStrategy) {
     // augment payload with inferred common terms
-    payload.preferences = gpii.matchMakerFramework.utils.addInferredCommonTerms(payload.preferences, payload.inferredCommonTerms);
+    payload.preferences = gpii.matchMakerFramework.utils.addInferredCommonTermsToPreferences(payload.preferences, payload.inferredCommonTerms);
+    // gpii.matchMakerFramework.utils.addCapabilitiesInformation(payload);
```

Comment:
Stray commented line - does this need to stay or go?

Reply:
I'll remove it.. This has now been moved to the matchmaker utitilies as part of building the MM input payload